src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/probationoffendersearch/ProbationOffenderSearchGatewayTest.kt

```diff
@@ -221,45 +328,46 @@ class ProbationOffenderSearchGatewayTest(
           }
         ]
         """,
-      )
+        )
 
-      val response = probationOffenderSearchGateway.getPerson(pncId)
+        val response = probationOffenderSearchGateway.getPerson(hmppsId)
 
-      response.data?.aliases.shouldBeEmpty()
-    }
+        response.data?.aliases.shouldBeEmpty()
+      }
 
-    it("returns null when 400 Bad Request is returned") {
-      probationOffenderSearchApiMockServer.stubPostOffenderSearch(
-        "{\"pncNumber\": \"$pncId\"}",
-        """
+      it("returns null when 400 Bad Request is returned") {
+        probationOffenderSearchApiMockServer.stubPostOffenderSearch(
+          "{\"crn\": \"$hmppsId\"}",
+          """
           {
             "developerMessage": "reason for bad request"
           }
           """,
-        HttpStatus.BAD_REQUEST,
-      )
-
-      val response = probationOffenderSearchGateway.getPerson(pncId)
-      response.data.shouldBeNull()
-      response.errors.shouldBe(
-        listOf(
-          UpstreamApiError(
-            causedBy = UpstreamApi.PROBATION_OFFENDER_SEARCH,
-            type = UpstreamApiError.Type.BAD_REQUEST,
+          HttpStatus.BAD_REQUEST,
+        )
+
+        val response = probationOffenderSearchGateway.getPerson(hmppsId)
+        response.data.shouldBeNull()
+        response.errors.shouldBe(
+          listOf(
+            UpstreamApiError(
+              causedBy = UpstreamApi.PROBATION_OFFENDER_SEARCH,
+              type = UpstreamApiError.Type.BAD_REQUEST,
+            ),
           ),
-        ),
-      )
-    }
+        )
+      }
 
-    it("returns null when no offenders are returned") {
-      probationOffenderSearchApiMockServer.stubPostOffenderSearch(
-        "{\"pncNumber\": \"$pncId\"}",
-        "[]",
-      )
+      it("returns null when no offenders are returned") {
+        probationOffenderSearchApiMockServer.stubPostOffenderSearch(
+          "{\"crn\": \"$hmppsId\"}",
+          "[]",
+        )
 
-      val response = probationOffenderSearchGateway.getPerson(pncId)
+        val response = probationOffenderSearchGateway.getPerson(hmppsId)
 
-      response.data.shouldBeNull()
+        response.data.shouldBeNull()
+      }
     }
```

Comment:
Tempted to put these new tests in different files so it's easy to clean up when we've made the switch

Reply:
We decided that having a `describe` block for the feature flag was enough.